src/Database/Query.php

```diff
@@ -1259,9 +1259,14 @@ public function orHaving($conditions, $types = [])
      * @param int|null $limit The number of rows you want in the page. If null
      *  the current limit clause will be used.
      * @return $this
+     * @throws \InvalidArgumentException If page number < 1.
      */
     public function page($num, $limit = null)
     {
+        if ($num < 1) {
+            $msg = 'Pages should start at 1.';
```

Comment:
Why use a variable instead of directly using the string on next line?

Reply:
if important I can change it

Reply:
The extra line is just pointless here.